src/components/error-dart.js

```diff
@@ -1,11 +1,71 @@
 import React from 'react'
+import axios from 'axios'
 import { Header, Button } from 'semantic-ui-react'
 
-import { getTheme, urlErrorAssets } from 'formula_one'
+import { getTheme, urlErrorAssets, urlWhoAmI } from 'formula_one'
 
 import blocks from '../css/error-dart.css'
+import { element } from 'prop-types'
 
 export default class ErrorDart extends React.PureComponent {
+  state = {
+    isFacultyOrGuest: false
+  }
+
+  isFacultyOrGuest = (roles) => {
+    let isFacultyOrGuest = false
+    roles.forEach(element => {
```

Comment:
Instead of a loop, you can use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).

Reply:
Actually we have a function ifRole which checks it. Refer omniport formula one documentation

Reply:
Used function ifRole, thank you for the suggestion